pkg/spec/createconfig.go

```diff
@@ -112,6 +112,7 @@ type NetworkConfig struct {
 type SecurityConfig struct {
 	CapAdd                  []string // cap-add
 	CapDrop                 []string // cap-drop
+	CapRequired             []string // cap-required
```

Comment:
Can we call it `CapBase` to be consistent with https://github.com/containers/libpod/pull/5209?

Reply:
I don't think this is the same thing.  CapBase is the default capabilities for all containers.